guides/common/modules/proc_setting-the-job-rate-limit-on-smartproxy.adoc

```diff
@@ -0,0 +1,20 @@
+[id="setting-the-job-rate-limit-on-{smart-proxy-context}_{context}"]
+= Setting the Job Rate Limit on {SmartProxy}
+
+You can limit the maximum number of active jobs on a {SmartProxy} at a time to prevent performance spikes.
+The job is active from the time {SmartProxy} first tries to notify the host about the job until the job is finished on the host.
+
+.Procedure
+. Run the `{foreman-installer}` command with the maximum number of jobs specified:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+{foreman-installer} --foreman-proxy-plugin-remote-execution-script-mqtt-rate-limit _MAX_JOBS_NUMBER_
+----
++
+For example:
```

Comment:
I am unsure about this example. Is 200 an somewhat sensible value? A recommendation by {Team}?

I'd probably completely drop it and instead mention the default value (maybe unlimited?).

Reply:
If value is sensible or not  depends on computing resources of the smartproxy. 200 should be as good as anything else. I'd avoid using a default value (if there is such, which I am not aware about) because  then there would be no point in setting it.

Reply:
The "default value" is no value, meaning unbounded. In this case, it is so highly dependent on the environment that there is no such thing as globally sensible value.

Reply:
I'm thinking "Note that the default value is 50" or "Note that by default, the number of jobs is unlimited". Maybe also add that this depends on the "compute power/network etc." of your {SmartProxy}.

Reply:
Added note :+1: